media/css/firefox/channel.less

```diff
@@ -4,195 +4,669 @@
+        padding-bottom: 32px;
+    }
 
-.pager-selected-firefox #firefox-link,
-.pager-selected-beta    #beta-link,
-.pager-selected-developer  #aurora-link {
-    top: 0;
-    left: 335px;
-    z-index: 3;
-}
+    @media only screen and (max-width: @breakDesktop) {
 
-.pager-selected-firefox #firefox-link a:hover,
-.pager-selected-beta    #beta-link    a:hover,
-.pager-selected-developer  #aurora-link  a:hover {
-    top: 0;
-}
+        h1 {
+            .font-size(44px);
+        }
+
+        h2 {
+            .font-size(24px);
+        }
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+
+        h1 {
+            .font-size(40px);
+        }
+
+        h2 {
+            .font-size(20px);
+        }
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
 
-/* To override default image styles */
+        h1 {
+            .background-size(100px 106px);
+            .font-size(36px);
+            padding-top: 150px;
+        }
 
-.pager-selected-firefox #beta-link,
-.pager-selected-developer  #beta-link {
-    top: 25px;
+        h2 {
+            .font-size(18px);
+        }
+    }
 }
 
-.pager-selected-firefox #beta-link img,
-.pager-selected-developer  #beta-link img {
-    width: 150px;
-    height: 150px;
+/*--------------------------------------------------------------------------*/
+// Platform navigation
+
+.platform-nav {
+    background: #1f366c;
+
+    ul {
+        margin: 0;
+        padding: 0;
+
+        li {
+            .font-size(24px);
+            .inline-block;
+            margin: 0 40px;
+            padding: @baseLine 0;
+
+            a:link,
+            a:visited {
+                .transition(color .1s ease-in-out);
+                color: #fff;
+            }
+
+            a:hover,
+            a:active,
+            a:focus {
+                .transition(color .1s ease-in-out);
+                color: @navActiveColor;
+                text-decoration: none;
+            }
+        }
+    }
+
+    @media only screen and (max-width: @breakDesktop) {
+
+        ul li {
+            .font-size(22px);
+        }
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+
+        .container {
+            width: 100%;
+        }
+
+        ul li {
+            .font-size(20px);
+            background: #dcdcdc;
+            border-top: 2px solid #fff;
+            display: block;
+            margin: 0;
+            padding: (@baseLine / 2) 0;
+            width: auto;
+
+            a:link,
+            a:visited {
+                color: @linkBlue;
+            }
+
+            a:hover,
+            a:active,
+            a:focus {
+                color: @linkBlueHover;
+            }
+        }
+    }
 }
 
-.pager-selected-firefox #firefox-link img,
-.pager-selected-beta    #beta-link img,
-.pager-selected-developer  #aurora-link img {
-    height: 225px;
-    width: 225px;
+html[lang^="en"] {
+    .platform-nav ul li {
+        text-transform: uppercase;
```

Comment:
This is in the design too, but I agree it looks both odd and probably not on-brand for Apple. I'll add a `:last-child` selector to make an exception for it.